Revamped Ultimate mode

dbryce712 opened the pull request:
-Midshield button changed to up taunt
-Mod X + Start = Select
-Mod Y + Start = Home
-Most analog values changed to PM mode values (some unchanged, e.g. shield tilt, firefox)

GrammyMoney reviewed (changes requested):
lgtm, need to update readme to include angle changes.

dbryce712 commented:
- Midshield button changed to up taunt
- Mod X + Start = Select
- Mod Y + Start = Home
- Samus can now input Homing Missiles with Mod X + side B
- Mod X + Up/Down + B now outputs Up/Down special, respectively
- Fixed issue where inputting DAFTs using Mod X + A outputted non-angled forward tilts instead
- Fixed issue that caused Mod X + C-Right to output a leftward-facing attack
- Horizontal Mod Y magnitude decreased to 35 from 41
- Mod Y Pivot Uptilt/Dtilt now functions properly
